test/benchmark/tests/utils/__init__.py

```diff
@@ -17,7 +18,7 @@
 
 DEFAULT_REGION = "us-west-2"
 IAM_INSTANCE_PROFILE = "EC2Admin"
-S3_BUCKET_BENCHMARK_ARTIFACTS = "s3://torchserve-model-serving/benchmark_artifacts"
+S3_BUCKET_BENCHMARK_ARTIFACTS = "s3://nikhilsk-model-serving/benchmark_artifacts"
```

Comment:
@nskool pls make sure we can pass this s3 bucket as an argument.

Reply:
Since this is a non-trivial and non-breaking change, I've added it to the issue which describes changes in phase 3 of the automation: https://github.com/pytorch/serve/issues/1121

Reply:
@nskool Why is the URL change to s3://torchserve-model-serving/xxx vs your personal S3 bucket a non-trivial change? Whatever tests we have can be put in the main s3://torchserve-model-serving bucket